### PR TITLE
more tests

### PR DESCRIPTION
**Pull request by GlazerMann**

### CODE REVIEW

File: tests/phpunit/TemplateTest.php [hunk 1]
```diff
@@ -182,6 +182,10 @@ public function testMisspeltParameters() {
     $this->assertNotNull($expanded->get('journal'));
     $this->assertNotNull($expanded->get('pages'));
     $this->assertNotNull($expanded->get('year'));
+      
+    $text = "{{Cite journal|pp. 1–5}}";
```
Comment (ms609) [inline tests/phpunit/TemplateTest.php]: Is this redundant to `testUseUnusedData` (line 36)?

File: tests/phpunit/TemplateTest.php [hunk 2]
```diff
@@ -204,4 +208,30 @@ public function testErrantAuthor() {
...
+    $this->assertNull($expanded->get('date'));
+    $this->assertEquals('1999',$expanded->get('year'));
+
+    $text = '{{cite journal|pmid=9858585|year=in press}}';
+    $expanded = $this->process_citation($text);
+    $this->assertNull($expanded->get('date'));
+    $this->assertEquals('1999',$expanded->get('year'));
+  }
+  
+  public function testSICI() {
```
Comment (ms609) [inline tests/phpunit/TemplateTest.php]: Does `testSICI` test anything that is not covered in `testSiciExtraction()` (line 146)?

File: tests/phpunit/TemplateTest.php [hunk 3]
```diff
@@ -182,6 +182,10 @@ public function testMisspeltParameters() {
     $this->assertNotNull($expanded->get('journal'));
     $this->assertNotNull($expanded->get('pages'));
     $this->assertNotNull($expanded->get('year'));
+      
+    $text = "{{Cite journal|pp. 1–5}}";
+    $expanded = $this->process_citation($text);
+    $this->assertNotNull($expanded->get('pages'));
```
Comment (ms609) [inline tests/phpunit/TemplateTest.php]: Is it worth using `AssertEquals('1–5', $expanded->get('pages');`?

Comment (GlazerMann) [reply tests/phpunit/TemplateTest.php]: Will do

File: tests/phpunit/TemplateTest.php (hunk 2 above)
Comment (GlazerMann) [reply tests/phpunit/TemplateTest.php]: Missed that one

File: tests/phpunit/TemplateTest.php (hunk 1 above)
Comment (GlazerMann) [reply tests/phpunit/TemplateTest.php]: This is for that bug where the first element gets deleted.  Don't want it coming back

**Pull request merged**